psi4/src/psi4/lib3index/cholesky.cc

```diff
@@ -163,7 +163,7 @@ CholeskyERI::CholeskyERI(std::shared_ptr<TwoBodyAOInt> integral, double schwarz,
     basisset_ = integral_->basis();
 }
 CholeskyERI::~CholeskyERI() {}
-size_t CholeskyERI::N() { return basisset_->nbf() * basisset_->nbf(); }
+size_t CholeskyERI::N() { return ( size_t ) basisset_->nbf() * basisset_->nbf(); }
```

Comment:
I think we're going with the `static_cast<type>(tgt)` style of casting. But can hold off b/c this type is changing more fundamentally in libmints.